projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -535,6 +535,12 @@ export interface FeatureTogglesInterface {
    */
   a11ySearchBoxFocusOnEscape?: boolean;
 
+  /**
+   * In `ItemCounterComponenet`, Remove button no longer lose focus after activating when count is 2.
+   * Add button no longer lose focus after activating when count is `max - 1`.
+   */
+  allyItemCounterFocus?: boolean;
```

Comment:
```suggestion
  a11yItemCounterFocus?: boolean;
```

Just for consistency's sake

Reply:
I've changed it.